External/Plugins/CodeRefactor/Provider/RefactoringHelper.cs

```diff
@@ -369,25 +370,13 @@ private static List<String> GetAllProjectRelatedFiles(IProject project, bool onl
             }
             else
             {
-                // NOTE: Could be simplified in .NET 3.5 with LINQ .Concat.Select.Distinct
-                var visited = new Dictionary<string, byte>();
-                foreach (string path in project.SourcePaths)
+                foreach (string path in project.SourcePaths.
```

Comment:
I prefer it this way, when I see a variable I think "this is used in more the one place", but I don't believe it matters that much nor my reasoning is any better, so I'll change it.